README.md

```diff
@@ -185,7 +185,17 @@ are joined to _test_mappings_ documents  _test_files_ field through the source_f
 	"test_mpapping_id" : ...,
 	"test_file_seen_count" : 2
 }
+
+### Analyze distrubution of thresholds
+The distribution of the thresholds for test_mappings and task_mappings can be
+seen by running the jupyter notebooks in the `notebooks` directory. To run
+them, set the SELECTED_TESTS_MONGO_URI environment variable to the database you
+would like to analyze and run jupyter notebooks:
+```
```

Comment:
maybe  add `pip install -r requirements-dev.txt ` (see my later comment).

Reply:
I would argue the setup steps should only be described once in the README, which happens here: https://github.com/mongodb/selected-tests#set-up-environment. Do you think this is clear enough, or should be rearranged?

Reply:
no, this is fine.